src/components/MultiSearch/index.tsx

```diff
@@ -350,6 +383,43 @@ export const MultiSearch: React.FC<MultiSearchProps> = ({
     return <Tag>{ftName}</Tag>;
   };
 
+  const actionsCreator = (item: any) => {
+    const feat = item.feature;
+    const layer = feat.get('layer');
+
+    if (!layer?.get('editable')) {
+      return;
+    }
+
+    const onEditFeatureBtnClick = () => {
+      dispatch(setLayerId(getUid(layer)));
+      dispatch(showEditFeatureDrawer());
+      setResultsVisible(false);
+    };
+
+    if (
+      allowedEditMode.includes('CREATE') ||
+      allowedEditMode.includes('DELETE') ||
+      allowedEditMode.includes('UPDATE')
+    ) {
```

Comment:
We should also check if the layer is marked as editable.

Reply:
It's checked already [here](https://github.com/terrestris/shogun-gis-client/pull/885/files/12858ce4a26c9006c0f9ec6d0860af61981a29dc#diff-2d094e5695965d7155bb73a7f9398c30472ceb99f1de5478fc74a29fa1fed5e0R390) in  `actionsCreator` and will return if its not editable.